docs/getting-started.txt

```diff
@@ -1,300 +1,338 @@
-#################
- Getting Started
-#################
+#########################
+ Quickstart for ML Users
+#########################
 
-This section walks through the steps to run examples in Determined from zero. You should be able to:
+This quickstart walks you through the basic steps to install and run Determined, train on the
+provided model data, and visualize training experiment results in your browser.
 
-#. Get a light touch with Determined concepts and features;
-#. Install Determined;
-#. Get familiar with the WebUI and the CLI;
-#. Train an example model with a single GPU;
-#. Train an example model with distributed training;
-#. Train an example model with hyperparameter tuning.
+Three model data and code examples use the MNIST dataset to demonstrate:
```

Comment:
Technically, all the examples use the same model code and the data. The only difference is the experiment configuration. So this sentence needs to be changed to be more precise.

Reply:
Added a clarifying sentence following the bullet items: "All three use the same model code and data, changing only a configuration parameter for different functionality."